ui/src/main/js/page/user/user/UserTable.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchUsers} from 'store/actions/users';
+import UserCopyRowAction from 'page/user/user/UserCopyRowAction';
+import UserEditRowAction from 'page/user/user/UserEditRowAction';
+import UserRoleCell from 'page/user/user/UserRoleCell';
+import Table from 'common/component/table/Table';
+import UserTableActions from 'page/user/user/UserTableActions';
+
+const COLUMNS = [{
+    key: 'username',
+    label: 'Username',
+    sortable: true
+}, {
+    key: 'emailAddress',
+    label: 'Email',
+    sortable: true
+}, {
+    key: 'authenticationType',
+    label: 'Authentication Type',
+    sortable: true
+}, {
+    key: 'roleNames',
+    label: 'Roles',
+    sortable: false,
+    customCell: UserRoleCell
+}, {
+    key: 'editUser',
+    label: 'Edit',
+    sortable: false,
+    customCell: UserEditRowAction,
+    settings: { alignment: 'center' }
+}, {
+    key: 'copyUser',
+    label: 'Copy',
+    sortable: false,
+    customCell: UserCopyRowAction,
+    settings: { alignment: 'center' }
+}]
+
+const UserTableConvert = ({canCreate, canDelete}) => {
```

Comment:
This is defined in the `UserTable.js` file.   The names should be consistent.  Should this component be named `UserTable` rather than `UserTableConvert` or should the file name be changed to `UserTableConvert.js`

Reply:
Good call - I've updated this to UserTable, as well as updated the BetaComponent section to identify the old table as `UserTableStagedForDelete`